skimage/measure/fit.py

```diff
@@ -293,8 +293,8 @@ def estimate(self, data):
         C, _, rank, _ = np.linalg.lstsq(A, f, rcond=None)
 
         if rank != 3:
-            warn("Input data does not contain enough significant data points. "
-                 "In scikit-image 1.0, this warning will become a ValueError.")
+            warn("Input data does not contain enough significant data points.")
```

Comment:
```suggestion
            warn("Input does not contain enough significant data points.")
```
(otherwise "input data do not ...")

Reply:
Oh, but then we need to update `scikit-image/skimage/measure/tests/test_fit.py::test_circle_model_insufficient_data` (line 163) @judafe

Reply:
@mkcor, I will do that today